constraint/pkg/client/regolib/src.go

```diff
@@ -38,6 +40,20 @@ audit[response] {
 	}
 }
 
+# get_default(data, "external", {}) seems to cause this error:
+# "rego_type_error: undefined function data.hooks.<target>.get_default"
+inventory[inv] {
+	inv = data.external["{{.Target}}"]
+}
+
+inventory[{}] {
```

Comment:
Nit: This rule is not be needed given the one below which asserts that if if data.external[<TARGET>] is missing, inv should be`{}`.

Reply:
My thought here is "what if data.external exists, but data.external.<target> does not.

Reply:
Oh, wait, I see what you mean. Updated.